tools/smartpeak/smartpeak_send_emails

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/env python
+from __future__ import print_function
+import pickle
+import os.path, os, sys
+from googleapiclient.discovery import build
+from google_auth_oauthlib.flow import InstalledAppFlow
+from google.auth.transport.requests import Request
+import base64, argparse
+from email.mime.multipart import MIMEMultipart
+from email.mime.text import MIMEText
+import markdown
+
+import smartpeak.utils as utils
+
+# If modifying these scopes, delete the file token.pickle.
+SCOPES = ['https://www.googleapis.com/auth/gmail.send']
+
+
+def create_message(sender, to, subject, msgHtml, msgPlain):
```

Comment:
Did you add in unit tests for these new methods?

Reply:
There is not really anything to test here, there two helper functions which only wraps message in MIME object and then content is encoded with base64. I would leave it as it is for now.